### PR TITLE
add auto-updater to Makefile

### PR DESCRIPTION
**Pull request by mniewrzal**
What: This change adds `auto-updater` binary to Make file binaries list.

Why: We want to have `auto-updater` binaries for all platforms.

Please describe the tests:
 - Test 1: none
 
Please describe the performance impact: none

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -260,8 +260,11 @@ versioncontrol_%:
 .PHONY: linksharing_%
 linksharing_%:
 	GOOS=$(word 2, $(subst _, ,$@)) GOARCH=$(word 3, $(subst _, ,$@)) COMPONENT=linksharing $(MAKE) binary
+.PHONY: auto-updater_%
+auto-updater_%:
+	GOOS=$(word 2, $(subst _, ,$@)) GOARCH=$(word 3, $(subst _, ,$@)) COMPONENT=auto-updater $(MAKE) binary	
 
-COMPONENTLIST := bootstrap certificates gateway identity inspector linksharing satellite storagenode uplink versioncontrol
+COMPONENTLIST := bootstrap certificates gateway identity inspector linksharing satellite storagenode uplink versioncontrol auto-updater
```
Comment (brimstone) [inline Makefile]: Put this in alphabetical order before `bootstrap` please.

Comment (mniewrzal) [reply Makefile]: done

**Comment by mniewrzal** [comment]
We had to rename binary, sorry for additional work.

**Comment by mniewrzal** [comment]
> I assume the autoupdater was renamed correctly in a different PR.

yes, exactly

**Pull request merged**